02week/pigLatin.js

```diff
@@ -9,8 +9,67 @@ const rl = readline.createInterface({
 
 
 function pigLatin(word) {
+  let word1 = word.toLowerCase().trim();
+  let newWord;
+  let vowels = ["a","e","i","o","u"];
+  //let vowls_w_y = ["a","e","i","o","u","y"];
+  let first_letter_vowel;
+  let first_vowel;
+  let midString = "";
+  let vowel_after1_yes = false;
 
-  // Your code here
+//loop to determine if there are any vowels after the first letter
```

Comment:
I really like how you have used console.log to print out what your code is doing.